db/migrate/20190715144518_create_backend_apis.rb

```diff
@@ -18,7 +18,7 @@ def up
     end
 
     create_table :backend_api_configs do |table|
-      table.string :path, null: false, default: ''
+      table.string :path, default: ''
```

Comment:
It is not ideal but otherwise it fails for Oracle 😑 
![image](https://user-images.githubusercontent.com/11318903/61968747-1107d200-afd9-11e9-8fba-b7dc2c9256bb.png)

Because for some reason it tries to enter `null` 😕 And this way, if we don't bypass the rails validation, it is correct and inserts `''`.
Gui showed me that the `path` is not inside `changed_attributes`, probably because `''` is the default value, but I don't understand that the same thing happens for MySQL and it works.


And with this change, running  for postgres `bundle exec rake db:drop db:create db:setup` with the same `schema` as in `master` with the idea of running the migrations after it:
![image](https://user-images.githubusercontent.com/11318903/61968841-51ffe680-afd9-11e9-9968-b6e70902dcf4.png)